src/main/scala/viper/silver/plugin/standard/termination/TerminationASTExtension.scala

```diff
@@ -52,7 +52,8 @@ case class DecreasesTuple(tupleExpressions: Seq[Exp] = Nil, override val conditi
 
   override lazy val prettyPrint: PrettyPrintPrimitives#Cont = {
     text("decreases") <>
-      (if (tupleExpressions.nonEmpty) space <> ssep(tupleExpressions map (toParenDoc(_)), char(',') <> space) else nil)
+      (if (tupleExpressions.nonEmpty) space <> ssep(tupleExpressions map (toParenDoc(_)), char(',') <> space) else nil) <>
```

Comment:
Nit picking -- I think the space there is unnecessary
```suggestion
      (if (tupleExpressions.nonEmpty) space <> ssep(tupleExpressions map (toParenDoc(_)), char(',')) else nil) <>
```

Reply:
I'd rather keep a space after commas in termination measure lists

Reply:
Oh I'm sorry! I misread the parentheses and thought this adds a space after the _list_ of comma separated `tupleExpressions` instead of after each element